crowvoorhees-key_west.md

```diff
@@ -0,0 +1 @@
+## new file
```

Comment:
@crowvoorhees Awesome! Is there anything you'd like to add to the file, maybe some fun things to do in your hometown or some good movies you've seen lately?